_config.yml

```diff
@@ -9,9 +9,22 @@ author:
 # Setup
 title: Elixir School
 description: Lessons in the Fundamentals of Elixir
-baseurl: '/'
-url: 'http://elixirschool.com' # the base hostname & protocol for your site
+baseurl: /
+url: https://elixirschool.com
```

Comment:
@3100 this configuration value is used in the generation of the language links.  I'll see if there's a way to assign values for development vs production in Jekyll.